src/gql/provider.tsx

```diff
@@ -34,8 +36,11 @@ const Provider: React.FC = ({ children }) => {
     if (!networkError && !graphQLErrors) return;
     const error = networkError?.message ?? graphQLErrors?.map(e => e.message).join(", ");
     store.dispatch(localSlice.actions.set({ error }));
+    error && reportError(error);
```

Comment:
```suggestion
    if (error) reportError(error);
```

Please use `&&` instead of `if` statement only in JSX.